configs/rover_configs.yaml

```diff
@@ -64,6 +64,37 @@ process_discovery:
         # The Process Labels, used to aggregate similar process from service entity
```

Comment:
One tip for `agent comment`

```
    agent:
      # Supports scanning for how long ago of the active agent
      process_status_refresh_period: ${ROVER_PROCESS_DISCOVERY_AGENT_PROCESS_STATUS_REFRESH_PERIOD:1m}
```

Change to `Set the agent refresh period`. Don't use an explanation sentence for the comment.

Reply:
Updated.